doc/setup/cdn.rst

```diff
@@ -37,7 +37,7 @@ Step 2: Configure the odoo instance with your zone
 
 In the Odoo back end, go to the :guilabel:`Website Settings`: menu, then
 activate the CDN support and copy/paste your zone URL in the
-:guilabel:`CDN Base URL` field.
+:guilabel:`CDN Base URL` field. This field is only visible and configurable if you have developer mode activated.
```

Comment:
that is not 81, that is exactly 114 characters 😉

Reply:
Hey, don't count the text that was already there. 😆 My part is 81 characters, being: " This field is only visible and configurable if you have developer mode activated."
Is it good for you now?